terraform/10-video-hearing-wowza/modules/wowza/network.tf

```diff
@@ -1,3 +1,11 @@
+data "azurerm_client_config" "current" {
+}
+locals {
+  dns_zone_name        = var.environment == "prod" ? "platform.hmcts.net" : "staging.platform.hmcts.net"
+  peering_vnets        = ["hmcts-hub-sbox-int"] #var.environment != "prod" && var.environment != "stg" ? ["hmcts-hub-prod-int", "ukw-hub-prod-int"] : []
```

Comment:
You are going to connect to the Staging DNS Zone, but peer to the SBox Hub?

Reply:
ouch, in my mind I was thinking stg was non-prod. I'v been jumping between subs too often